proto/streamlit/proto/Image.proto

```diff
@@ -36,11 +36,15 @@ message Image {
 message ImageList {
   repeated Image imgs = 1;
 
+  // @see WidthBehaviour on the backend
+  // @see WidthBehavior on the frontend
   // The width of each image.
   // >0 sets the image width explicitly
```

Comment:
not in the scope of this PR, and maybe i'll see it when reviewing the rest of the code, but is this a percentage, or pixels?

Reply:
Thanks for reviewing, but I need to fix a few bugs first (this is still a draft)! But to answer your question, this is in pixels

Reply:
pixels